src/Microsoft.ML.CodeGenerator/Templates/Console/ModelProject.tt

```diff
@@ -9,22 +9,25 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="1.4.0" />
+    <PackageReference Include="Microsoft.ML" Version="<#= StablePackageVersion #>" />
 <# if (IncludeLightGBMPackage) { #>
-    <PackageReference Include="Microsoft.ML.LightGBM" Version="1.4.0" />
+    <PackageReference Include="Microsoft.ML.LightGBM" Version="<#= StablePackageVersion #>" />
 <#}#>
 <# if (IncludeMklComponentsPackage){ #>
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.4.0" />
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="<#= StablePackageVersion #>" />
 <#}#>
 <# if (IncludeFastTreePackage){ #>
-    <PackageReference Include="Microsoft.ML.FastTree" Version="1.4.0" />
+    <PackageReference Include="Microsoft.ML.FastTree" Version="<#= StablePackageVersion #>" />
 <#}#>
 <# if (IncludeImageTransformerPackage){ #>
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.4.0" />
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="<#= StablePackageVersion #>" />
 <#}#>
 <# if (IncludeImageClassificationPackage){ #>
-    <PackageReference Include="Microsoft.ML.Dnn" Version="1.4.0" />
-	<PackageReference Include="SciSharp.TensorFlow.Redist" Version="1.14.0" />
+    <PackageReference Include="Microsoft.ML.Dnn" Version="<#= StablePackageVersion #>" />
+	<PackageReference Include="SciSharp.TensorFlow.Redist" Version="<#= StablePackageVersion #>" />
```

Comment:
Can you fix the whitespace (there's a tab at the beginning) while you are in here?